packages/core/src/operator/checkRefereeBulkSubmissionCompatible.ts

```diff
@@ -0,0 +1,57 @@
+import { ethers } from 'ethers';
+import { getCurrentRefereeVersionFromGraph, RefereeAbi } from '../index.js';
+import { config } from '../config.js';
+import { RefereeConfig } from "@sentry/sentry-subgraph-client";
+import { getProvider } from "../index.js";
+
+/**
+ * Checks if the current referee contract is compatible with bulk submission.
+ * 
+ * @param {boolean} graphIsHealthy - Indicates whether the graph is healthy.
+ * @param {RefereeConfig | undefined} refereeConfig - The configuration object for the referee, if already retrieved.
+ * @returns {Promise<boolean>} - Returns `true` if the referee contract is compatible with bulk submission, otherwise `false`.
+ */
+export async function checkRefereeBulkSubmissionCompatible(
+    graphIsHealthy: boolean,
+    refereeConfig: RefereeConfig | undefined,
+): Promise<boolean> {
+
+    // If a refereeConfig is provided, check its version
+    if (refereeConfig) {
+        // If the version is greater than 6, it is compatible with bulk submission
+        if (refereeConfig.version > 6) {
+            return true;
+        }
+    } else {
+        // If no refereeConfig is provided, decide based on the graph health
+        if(graphIsHealthy) {
+            // If the graph is healthy, load the referee version from the graph
+            const refereeVersion: number = await getCurrentRefereeVersionFromGraph();
+            // If the version is greater than 6, it is compatible with bulk submission
+            if (refereeVersion > 6) {
+                return true;
+            }
```

Comment:
I think the `graphIsHealthy` is redundant.

If `refereeConfig` is defined, we should asume that the subgraph is healthy and we have the current config.
If its undefined we should query from RPC.

Let me know if that makes sense to you, I might have miscommunicated that in our call

Reply:
This has been updated to remove the redundant graph check! Good catch!